packages/shortest/package.json

```diff
@@ -49,6 +49,7 @@
     "commander": "^13.1.0",
     "glob": "^10.4.5",
     "globby": "^14.1.0",
+    "listr2": "^8.2.5",
```

Comment:
Picked this one as it can be used with [`@inquirer/prompts`](https://www.npmjs.com/package/@inquirer/prompts) via an [adapter](https://github.com/listr2/listr2/tree/master/packages/prompt-adapter-inquirer)